examples/tv-casting-app/linux/simple-app-helper.cpp

```diff
@@ -256,23 +274,49 @@ void ConnectionHandler(CHIP_ERROR err, matter::casting::core::CastingPlayer * ca
             "simple-app-helper.cpp::ConnectionHandler(): Failed to connect to CastingPlayer (ID: %s) with err %" CHIP_ERROR_FORMAT,
             targetCastingPlayer->GetId(), err.Format()));
 
-    ChipLogProgress(AppServer, "simple-app-helper.cpp::ConnectionHandler(): Successfully connected to CastingPlayer (ID: %s)",
-                    castingPlayer->GetId());
-    ChipLogProgress(AppServer,
-                    "simple-app-helper.cpp::ConnectionHandler(): Triggering demo interactions with CastingPlayer (ID: %s)",
-                    castingPlayer->GetId());
+    if (gCommissionerGeneratedPasscodeFlow)
+    {
+        ChipLogProgress(AppServer,
+                        "simple-app-helper.cpp::ConnectionHandler(): Successfully connected to CastingPlayer (ID: %s) using "
+                        "Commissioner-Generated passcode",
+                        castingPlayer->GetId());
+        ChipLogProgress(AppServer, "simple-app-helper.cpp::ConnectionHandler(): Desired Endpoint ID for demo interactions: 1");
+    }
+    else
+    {
+        ChipLogProgress(AppServer, "simple-app-helper.cpp::ConnectionHandler(): Successfully connected to CastingPlayer (ID: %s)",
+                        castingPlayer->GetId());
+        ChipLogProgress(AppServer,
+                        "simple-app-helper.cpp::ConnectionHandler(): Desired Endpoint Vendor ID for demo interactions: %d",
+                        kDesiredEndpointVendorId);
+    }
 
+    ChipLogProgress(AppServer, "simple-app-helper.cpp::ConnectionHandler(): Getting endpoints avaiable for demo interactions");
     std::vector<matter::casting::memory::Strong<matter::casting::core::Endpoint>> endpoints = castingPlayer->GetEndpoints();
+    PrintEndpointsVendorAndProductIDs(endpoints);
+
     // Find the desired Endpoint and auto-trigger some Matter Casting demo interactions
     auto it = std::find_if(endpoints.begin(), endpoints.end(),
                            [](const matter::casting::memory::Strong<matter::casting::core::Endpoint> & endpoint) {
-                               return endpoint->GetVendorId() == kDesiredEndpointVendorId;
+                               if (gCommissionerGeneratedPasscodeFlow)
+                               {
+                                   // Since the tv-app with Vendor ID 1111 does not implement the endpoint with Vendor ID 65521 we
+                                   // will (temporarily) use the first available endpoint for demo interactions. See
+                                   // connectedhomeip/examples/tv-app/tv-common/include/AppTv.h.
```

Comment:
```suggestion
                                   // Since the tv-app with Vendor ID 1111 does not implement the endpoint with Vendor ID 65521 we
                                   // will (temporarily) use the endpoint with ID 1 for demo interactions. See
                                   // connectedhomeip/examples/tv-app/tv-common/include/AppTv.h.
```